frontend/.env

```diff
@@ -1 +1 @@
-REACT_APP_API="https://hitk-community-website-backend.herokuapp.com"
+REACT_APP_API="http://localhost:3500"
```

Comment:
Hey @Anurag-Singh-Rajput the backend of this website is hosted at this location only so can you please change back to the heroku one. It should work fine.

Reply:
Ok @Kajol-Kumari , Actually the backend doesn't seem to return any json data when i tried the route https://hitk-community-website-backend.herokuapp.com/broadcast 
 I will look into it and make the necessary changes.

Reply:
Hey @Kajol-Kumari ,
I think I found the problem. I tried to deploy same backend code to heroku and i was able to access /broadcast route.
I think there might be one of these problems, please check on Heroku(where backend is deployed and probably connected to GitHub):
1.) Is the Auto-deploy turned on?
2.) Is the branch selected in deploy is "Master"?

Reply:
Thanks for pointing this @Anurag-Singh-Rajput I checked the hosted-backend and I found issue with the auto-deploy, I fixed that and now the bug seems to be resolved. 

As you can see the data isn't getting populated on the cards of the broadcast page, can you look into the same?

Reply:
@Kajol-Kumari  Should I add program name under "Brief Description of what is Fixed or changed ' ?

Reply:
Yes, this works we just need to know under which program this PR has been created in order to keep the track :)

Reply:
Great Thanks ! This was my first PR :)